File: solutions/uri/1036/1036.c
#include <math.h>
#include <stdio.h>

int16_t main() {
    double a, b, c, delta;

    scanf("%lf %lf %lf", &a, &b, &c);

    delta = (b * b) - 4 * a * c;
    if (delta >= 0 && a != 0) {
        printf("R1 = %.5lf\n", ((b * -1) + sqrt(delta)) / (2 * a));
        printf("R2 = %.5lf\n", ((b * -1) - sqrt(delta)) / (2 * a));
    } else {
        printf("Impossivel calcular\n");
    }

    return 0;
}

File: solutions/uri/1036/1036.py
import math

a, b, c = map(float, input().split())

delta = math.pow(b, 2) - 4 * a * c

if delta >= 0 and a != 0:
    print(f'R1 = {((b * -1) + math.sqrt(delta)) / (2 * a):.5f}')
    print(f'R2 = {((b * -1) - math.sqrt(delta)) / (2 * a):.5f}')
else:
    print('Impossivel calcular')
